apps/files_sharing/src/views/SharingDetailsTab.vue

```diff
@@ -727,15 +735,24 @@ export default {
 	beforeMount() {
 		this.initializePermissions()
 		this.initializeAttributes()
-		console.debug('shareSentIn', this.share)
-		console.debug('config', this.config)
+		this.logger.debug('shareSentIn', this.share)
+		this.logger.debug('config', this.config)
 	},
 
 	mounted() {
 		this.$refs.quickPermissions?.querySelector('input:checked')?.focus()
 	},
 
 	methods: {
+		async getNode() {
+			const node = { path: this.path }
+			try {
+				this.node = await fetchNode(node)
+				this.logger.info('Fetched Node:', this.node)
+			} catch (error) {
+				this.logger.error('Error:', error)
```

Comment:
This is even custom formatting for errors in the logger if you just pass it like:
```suggestion
				this.logger.error(error)
```